src/Views/AppInfoView.vala

```diff
@@ -313,6 +323,45 @@ namespace AppCenter.Views {
             });
         }
 
+        private void load_releases () {
```

Comment:
If you're going to make a new class for ReleaseRow, maybe this function could be refactored into it's own class, don't you think? :) 

Because one idea was to also have the Releases in the Updates tab, so maybe this could get reused in the future